### DISCUSSION TITLE
Multiple Connections / Databases / Datasources

### DISCUSSION THREAD

**Discussion by svenjia** [prisma/prisma#17855] [Q&A]
I have a project using Mongodb and Mysql both. Its really a pain for useing Knex.js(sql builder) and Mongoose (mongodb ORM). So,is prisma can be used for the both. if not,how can i design my .env file and prisma.schema file for mutiple database connections? Thank you guys~

**Comment by nurul3101** [comment]
Hi @svenjia 👋 

I would recommend you to have a look at this GitHub Issue
- https://github.com/prisma/prisma/issues/2443

And particular this [comment](https://github.com/prisma/prisma/issues/2443#issuecomment-630679118) demonstrates how you can have multiple PrismaClients and multiple schema files that connect to different databases.

**Comment by svenjia** [comment]
Thank you for your reply! @nurul3101 . I have read that Issue,it seems still dont have an official illustration or some examples in your official docs. wish you guys can update your docs obout how to use prisma for mutiple type of dbs (RDB & NoSQL) and mutiple connections of dbs.
Appreciate your effects

**Reply by nikolasburk** [reply]
Hey @svenjia, this is definitely on our radar for improving the docs! We have an open issue for it here already: https://github.com/prisma/docs/issues/1940

Do you have any _concrete_ questions that should be addressed in such docs? (In general, the mental model for Prisma doesn't really change when using multiple `PrismaClient` instances where each one connects to a DB, so I'd be curious if you have any specific concerns/questions about this topic 🙂 )

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
